src/components/editor/EditorAreaMenu.tsx

```diff
@@ -188,11 +209,16 @@ export default function Banner(props: { [x: string]: any }) {
 					_focus={{
 						bg: 'transparent'
 					}}
-					mb='10px'>
+					mb='10px'
+					onClick={() => {
```

Comment:
Ideally, handleHTMLSave should be this entire function, and no logic should be in this prop.

Reply:
Got it, will do that!